android-sdk/src/main/java/webtrekk/android/sdk/util/Util.kt

```diff
@@ -65,4 +66,16 @@ val currentTimeStamp: Long
     inline get() = System.currentTimeMillis()
 
 val currentWebtrekkVersion: String
-    inline get() = BuildConfig.VERSION_NAME
\ No newline at end of file
+    inline get() = BuildConfig.VERSION_NAME
+
+fun jsonToMap(params: String): Map<String, String> {
+    val obj = JSONObject(params)
```

Comment:
`JSONObject` may throw `JSONException`. We need to catch that to prevent crashes.